lib/openers/atompdf-opener.js

```diff
@@ -5,18 +5,27 @@ import Opener from '../opener'
 export default class AtomPdfOpener extends Opener {
   open (filePath, texPath, lineNumber, callback) {
     // Opens PDF in a new pane -- requires pdf-view module
-    const openPanes = atom.workspace.getPaneItems()
-    for (const openPane of openPanes) {
+
+    function forwardSync(pdfView) {
+      if (pdfView != null && pdfView.forwardSync != null) {
+        pdfView.forwardSync(texPath, lineNumber)
+      }
+    }
+
+    const openPaneItems = atom.workspace.getPaneItems()
+    for (const openPaneItem of openPaneItems) {
       // File is already open in another pane
-      if (openPane.filePath === filePath) { return }
+      if (openPaneItem.filePath === filePath) {
+        forwardSync(openPaneItem)
+        return
+      }
     }
 
-    const pane = atom.workspace.getActivePane()
+    const activePane = atom.workspace.getActivePane()
     // TODO: Make this configurable?
-    // FIXME: Migrate to Pane::splitRight.
-    const newPane = pane.split('horizontal', 'after')
-    // FIXME: Use public API instead.
-    atom.workspace.openURIInPane(filePath, newPane)
+    atom.workspace.open(filePath, {'split': 'right'}).done(function (pdfView) {
+      forwardSync(pdfView)
+    })
```

Comment:
The inline function is unnecessary since you've defined the `forwardSync` function above. Replace with
```js
atom.workspace.open(filePath, {'split': 'right'}).done(forwardSync)
```